Package.swift

```diff
@@ -150,7 +150,7 @@ let package = Package(
       url: "https://github.com/google/GoogleAppMeasurement.git",
       // Note that CI changes the version to the head of main for CI.
       // See scripts/setup_spm_tests.sh.
-      .exact("10.4.0")
+      .exact("10.6.0")
```

Comment:
I _think_ you'll need to add a tag named `10.6.0` in the [GoogleAppMeasurement](https://github.com/google/GoogleAppMeasurement.git) repo before we merge this change (so that Swift Package Manager can find it).

Reply:
That doesn't happen until release day. Until then, CI builds the `main` branch

Reply:
Thanks for the clarification that it's known to break non-CI builds targeting master.

Linking to the issue / possible workaround you shared: https://github.com/google/GoogleAppMeasurement/issues/24#issuecomment-1097109812

Reply:
Getting the error in the firebase-ios-sdk:
 
Failed to resolve dependencies Dependencies could not be resolved because no versions of 'googleappmeasurement' match the requirement 10.6.0 and 'firebase-ios-sdk' depends on 'googleappmeasurement' 10.6.0.

The public doesn't have access to  https://dl.google.com/firebase/ios/analytics/93d139ae9f688e9c/GoogleAppMeasurement-10.6.0.tar.gz

Reply:
That's expected behavior. 10.6.0 has not yet released.